add-hops/docker-compose.yml

```diff
@@ -0,0 +1,145 @@
+version: '2'
+
+services:
+
+  baseline_app1:
+    build: .
+    command: -downstream=http://frontend:8502
+
+  baseline_app2:
+    build: .
+    command: -downstream=http://frontend:8502
+
+  baseline_app3:
+    build: .
+    command: -downstream=http://frontend:8502
+
+  baseline_app4:
+    build: .
+    command: -downstream=http://frontend:8502
+
+  baseline_app5:
+    build: .
+    command: -downstream=http://frontend:8502
+
+  baseline_app6:
+    build: .
+    command: -downstream=http://frontend:8502
+
+  baseline_app7:
+    build: .
+    command: -downstream=http://frontend:8502
+
+  baseline_app8:
+    build: .
+    command: -downstream=http://frontend:8502
+
+  baseline_app9:
+    build: .
+    command: -downstream=http://frontend:8502 -latency=1s
+
+  baseline_app10:
+    build: .
+    command: -downstream=http://frontend:8502 -latency=2s -success-rate=0.6
+
+  linkerd_app1:
+    build: .
+    command: -downstream=http://frontend:8503
+
+  linkerd_app2:
+    build: .
+    command: -downstream=http://frontend:8503
+
+  linkerd_app3:
+    build: .
+    command: -downstream=http://frontend:8503
+
+  linkerd_app4:
+    build: .
+    command: -downstream=http://frontend:8503
+
+  linkerd_app5:
+    build: .
+    command: -downstream=http://frontend:8503
+
+  linkerd_app6:
+    build: .
+    command: -downstream=http://frontend:8503
+
+  linkerd_app7:
+    build: .
+    command: -downstream=http://frontend:8503
+
+  linkerd_app8:
+    build: .
+    command: -downstream=http://frontend:8503
+
+  linkerd_app9:
+    build: .
+    command: -downstream=http://frontend:8503 -latency=1s
+
+  linkerd_app10:
+    build: .
+    command: -downstream=http://frontend:8503 -latency=2s -success-rate=0.6
+
+  linkerd:
+    image: buoyantio/linkerd:0.8.6
+    ports:
+      - 4140:4140
+      - 9990:9990
+    volumes:
+      - ./linkerd.yml:/io/buoyant/linkerd/config.yml:ro
+      - ./disco:/disco
+    command:
+      - "/io/buoyant/linkerd/config.yml"
+
+  frontend:
+    image: library/nginx:1.10
+    ports:
+      - 8502:8502
+      - 8503:8503
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf:ro
+    depends_on:
+      - baseline_app1
+      - baseline_app2
+      - baseline_app3
+      - baseline_app4
+      - baseline_app5
+      - baseline_app6
+      - baseline_app7
+      - baseline_app8
+      - baseline_app9
+      - baseline_app10
+
+  prometheus:
+    image: prom/prometheus:v1.4.1
+    volumes:
+      - ./prometheus.yml:/prometheus/prometheus.yml:ro
+    command: >
+      -config.file=/prometheus/prometheus.yml
+      -storage.local.retention=12h
+    ports:
+      - 9090:9090
+
+  grafana:
+    build:
+      context: .
+      dockerfile: Dockerfile-grafana
+    ports:
+      - 3000:3000
+    environment:
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin # TODO: need this to add data source, but causes "getting started" panel on dashboard
```

Comment:
Interesting -- I wonder why I didn't run into this with the failure-accrual demo.